scripts/check-autodoc.py

```diff
@@ -70,5 +70,7 @@ def extract_folder_names(test_dir: Path) -> Tuple[Path, str]:
 
 if len(missing_funcs) > 0:
     raise Exception(
-        f"The following functions have not yet been added: {missing_funcs}"
+        f'''The following functions have not yet been added: {missing_funcs}.
+         Please add the function names to docs/reference/general_functions.rst
+         under an appropriate section.'''
```

Comment:
Black convention usually would suggest using the double quotes for docstrings:

```suggestion
        f"""The following functions have not yet been added: {missing_funcs}.
         Please add the function names to docs/reference/general_functions.rst
         under an appropriate section."""
```

Reply:
I've replaced the single quotes by double quotes. It seems the auto styling does not run over the scripts so I missed it.

Reply:
No worries! Looks like that's something for me to take care of.